docker_utils/rtenvsub.sh

```diff
@@ -20,10 +20,82 @@ version='0.1'
 # Set a safe umask
 umask 0077
 
-# Import the standard shell library
-# shellcheck source=../shtdlib.sh disable=SC1091
-source "$(dirname "${0}")/../shtdlib.sh" &> /dev/null || ../shtdlib.sh &> /dev/null || source ./shtdlib.sh &> /dev/null || source shtdlib.sh
+default_library_name='shtdlib.sh'
+default_base_download_url='https://raw.githubusercontent.com/sdelements/shtdlib/master'
+default_install_path='/usr/local/bin'
+
+# Temporary debug function
+type -t import | grep -q '^function$' || function debug { echo "${@:2}" ; }
+
+# Import or source
+function import_or_source {
+    if type -t import | grep -q '^function$' ; then
+        debug 10 "Importing ${0}"
```

Comment:
Debug and Import use different variables in both the `if` and `else` block

Reply:
I thought that was fixed, odd.